api4/job.go

```diff
@@ -70,6 +70,11 @@ func downloadJob(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	if job.Type == model.JOB_TYPE_MESSAGE_EXPORT && !c.App.SessionHasPermissionTo(*c.App.Session(), model.PERMISSION_SYSCONSOLE_READ_COMPLIANCE) {
+		c.SetPermissionError(model.PERMISSION_SYSCONSOLE_READ_COMPLIANCE)
+		return
+	}
```

Comment:
This should be moved above call to `GetJob`.

Reply:
Hey @sbishel !

Not sure if I fully understand but if you meant put that statement above the GetJob call, I wanted too however I don't have access to the job.type property at that point which is why I put it after :slightly_frowning_face: We only have Job ID before then and we use that to get the job. Unless I've misunderstood your meaning 😅

Reply:
You're right. Thanks for explanation.